### PR TITLE
Removes the tsxs command line tool

### PR DESCRIPTION
**Pull request by zwoop**

### CODE REVIEW

**Comment by zwoop** [comment]
[approve ci autest]

**Review by dragon512 — APPROVED** [review]

File: src/traffic_layout/file_system.cc [hunk 1]
```diff
@@ -41,17 +41,9 @@ static std::string remove_path;
...
-                                           "traffic_logstats",
-                                           "traffic_server",
-                                           "traffic_top",
-                                           "traffic_via",
-                                           "trafficserver",
-                                           "tspush",
-                                           "tsxs"};
+std::set<std::string> const executables = {"traffic_crashlog", "traffic_ctl",    "traffic_layout", "traffic_logcat",
+                                           "traffic_logstats", "traffic_server", "traffic_top",    "traffic_via",
+                                           "trafficserver",    "tspush"};
```
Comment (traeak) [inline src/traffic_layout/file_system.cc:46]: put a comma after "tspush" to fix up formatting.

Comment (zwoop) [reply src/traffic_layout/file_system.cc:46]: The "improvements" with the extra comma are pretty insignificant, it adds an empty line after the '{', and pushes things to the left. 

```
+std::set<std::string> const executables = {
+  "traffic_crashlog", "traffic_ctl", "traffic_layout", "traffic_logcat", "traffic_logstats",
+  "traffic_server",   "traffic_top", "traffic_via",    "trafficserver",  "tspush",
+};
```

**Comment by zwoop** [comment]
We need to wait a couple of days to merge this though, to make sure there are no -1's.

File: tests/gold_tests/autest-site/setup.cli.ext [hunk 2]
```diff
@@ -123,22 +122,13 @@ if ENV['ATS_BIN'] is not None:
     Variables.trafficserver_version = out
     host.WriteVerbose(['ats'], "Traffic server version:", out)
 
-    # this queries tsxs for build flags so we can build code for the tests and get certain
-    # useful flags as which openssl to use when we don't use the system version
+    # ToDo: This used to use tsxs to find some libraries, maybe it needs to use cmake tools?
```
Comment (bneradt) [inline tests/gold_tests/autest-site/setup.cli.ext:125]: Let's just remove this TODO. Now that our tools are just built with cmake itself, I think this isn't applicable anymore.

The rest of this PR looks fine. Poke me when you update and I'll approved.

**Review by bneradt — CHANGES_REQUESTED** [review]

**Review by bneradt — APPROVED** [review]

**Pull request merged**